docs/4.0.0/404.html

```diff
@@ -12,15 +12,15 @@
       
       
       <link rel="icon" href="/IGUANA_logo.png">
-      <meta name="generator" content="mkdocs-1.6.0, mkdocs-material-9.5.31">
+      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.15">
```

Comment:
this looks wrong

Reply:
Yeah, I generated it locally, I used the wrong version